benchbuild/utils/compiler.py

```diff
@@ -18,13 +18,17 @@
 Are just convencience methods that can be used when interacting with the
 configured llvm/clang source directories.
 """
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)